app/Actions/Diagnostics/Pipes/Checks/AppUrlMatchCheck.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Actions\Diagnostics\Pipes\Checks;
+
+use App\Contracts\DiagnosticPipe;
+
+class AppUrlMatchCheck implements DiagnosticPipe
+{
+	/**
+	 * {@inheritDoc}
+	 */
+	public function handle(array &$data, \Closure $next): array
+	{
+		$config_url = config('app.url');
+		// http:// is 7 characters.
```

Comment:
```suggestion
		// https:// is 8 characters.
```
As we encourage https it may be best to specify that in this comment? We are meant to put the https there, right? (genuine question..)
A quick look at the [php `strpos` docs](https://www.php.net/manual/en/function.strpos.php) suggests the test below still covers this case anyway.

Reply:
I tested with https:// :')